src/app/lib/config/services/pidgin.js

```diff
@@ -115,6 +115,20 @@ const pidgin = {
       audio: 'Audio',
       video: 'Video',
     },
+    durations: {
+      seconds: {
+        singular: 'sekunda',
+        plural: 'sekund',
+      },
+      minutes: {
+        singular: 'minuta',
+        plural: 'minut',
+      },
+      hours: {
+        singular: 'godzina',
+        plural: 'godzin',
+      },
+    },
```

Comment:
this isn't actually pidgin. I used Polish translations because Pidgin doesn't exist on Google Translate